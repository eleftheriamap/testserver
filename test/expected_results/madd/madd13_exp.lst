
test/exp/test/gen/madd/madd13_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	51399884 	sub	w4, w4, #0xe66
   4:	d1151c00 	sub	x0, x0, #0x547
   8:	912bd929 	add	x9, x9, #0xaf6
   c:	9b002483 	madd	x3, x4, x0, x9
  10:	8a000000 	and	x0, x0, x0
