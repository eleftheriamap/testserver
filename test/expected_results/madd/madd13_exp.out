Registers:
X00    = fffffffffffffab9
X01    = 0000000000000000
X02    = 0000000000000000
X03    = fffffab9004c0740
X04    = 00000000fffff19a
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000af6
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x51399884
0x00000004 : 0xd1151c00
0x00000008 : 0x912bd929
0x0000000c : 0x9b002483
0x00000010 : 0x8a000000
